README.md

```diff
@@ -6,24 +6,25 @@ This module is the Web Interface part of the Zonemaster project.
 
 ### Installation
 
-Follow the detailed [installation instructions](docs/Installation.md).
+Follow the detailed [installation instructions].
 
 ##### Prerequisites
-Before you install the Zonemaster Web GUI module, you need the
-Zonemaster Engine test framework installed. Please see the
-[Zonemaster Engine installation instructions](https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Installation.md)
 
-And also the the zonemaster-backend module installed. Please see the [Zonemaster
-Backend installation](https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Installation.md)
+Before you install the Zonemaster Web GUI module, you need the Zonemaster
+Engine test framework installed. Please see the [Zonemaster-Engine
+installation] document.
+
+And also the the zonemaster-backend module installed. Please see the
+[Zonemaster-Backend installation] document.
 
 
 ##### Configuration
 
-Texts for configuring the backend are found in the [installation
-instructions](docs/Installation.md).
+The configuration instructions for the backend can be found in the [Backend
+configuration] document.
 
-The configuration instructions for the GUI can be found in the [configuration
-documentation](docs/Configuration.md).
+The configuration instructions for the GUI can be found in the [GUI
+configuration] document.
 
 
 ### Contribution
```

Comment:
Further down I find the following that need rework. 
```
### Documentation
Basically, the GUI is a serverless Angular application which use the Backend JSONRC API.
Other than the installation file, the [docs directory](docs/), contains a translation guide for the GUI
```

Reply:
> Further down I find the following that need rework.

I think the whole document needs an update. Some part are 4 years old and are outdated. Would it be ok to leave this to a later work?

Reply:
Well, it says that the installation document is in the docs directory, which was true before the move, but not true anymore.